src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -76,18 +79,16 @@ public void testGetUser_byId_notFound() {
 
   @Test
   public void testAddUser() {
-    User inputUser =
```

Comment:
If you keep the original `addUser` as well, then you'll need to keep this test method as well, and rename the new one (`testAddUserByName` ?) so they don't have the same name.